.gitignore

```diff
@@ -30,3 +30,10 @@
 *.exe
 *.out
 *.app
+
+# build
+builddir
```

Comment:
This is not necessary, since Meson already creates `builddir/.gitignore` for you with the contents `*`.

Meson knows when it creates the builddir that nothing in it should ever get checked into git, so it is easy to do the right thing and gitignore it all for you. :)